packages/stegoveritas/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stegoveritas
+pkgver=1.1
+pkgrel=1
+pkgdesc='Automatic image steganography analysis tool'
+arch=('any')
+groups=('blackarch' 'blackarch-stego')
+url='https://github.com/bannsec/stegoVeritas/releases'
+license=('MIT')
+depends=('python' 'python-pillow' 'python-numpy' 'python-python-magic' 'python-prettytable' 'python-exifread' 'python-python-xmp-toolkit' 'python-stegoveritas-binwalk' 'python-pypng' 'python-apng' 'perl-image-exiftool' 'p7zip' 'foremost')
```

Comment:
Try to indent better (no all in one line)

Reply:
ok